policyengine_us/variables/gov/states/co/tax/income/credits/eitc/co_eitc_eligible.py

```diff
@@ -0,0 +1,15 @@
+from policyengine_us.model_api import *
+
+
+class co_eitc_eligible(Variable):
```

Comment:
I don't think we need this variable. We can just set `co_eitc`'s `defined_for = StateCode.CO`.